Makefile

```diff
@@ -5,13 +5,14 @@ PWD := $(shell pwd)
 # Determine OS & Arch for specific OS only tools on Unix based systems
 OS := $(shell uname | tr '[:upper:]' '[:lower:]')
 ifeq ($(OS),darwin)
-	SED := gsed
+	SED ?= gsed
```

Comment:
I don't have `gsed` in my environment. By doing this I can pass my sed command via the command line. Hopefully this is ok with you.